packages/affine/model/src/consts/line.ts

```diff
@@ -0,0 +1,37 @@
+import { createZodUnion } from '../utils/zod.js';
+
+export enum LineColor {
```

Comment:
Should we use `@toeverything/theme` and `cssVarV2` as mentioned in the weekly meeting?

Reply:
![截屏2024-08-22 10.20.03.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/sJGviKxfE3Ap685cl5bj/21ffd9c7-4285-40da-8e88-4da7fa669114.png)

I'm afraid not. This `--affine-xxx-xxx` is saved into data model.

Reply:
I think it's better to save `yellow` rather than `--affine-palette-line-yellow` in datastrcuture.

Reply:
He suggested storing real values. /cc @EYHN

Reply:
@fundon Maybe we can use a color system like `@toeverything/theme` and turn `--affine-palette-shape-yellow`, `--affine-palette-line-yellow` and `--affine-palette-shape-yellow`in to the same `theme-yellow` variable. But legacy data is a barrier to this refactoring. @Saul-Mirone

Reply:
It makes sense to name them separately, so that we can adjust the shape's border and background, as well as the note's background, separately.

If we migrate to real color values in the future, we can avoid the problems associated with theme upgrades.